SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.cpp

```diff
@@ -495,6 +511,12 @@ void GenerateSubobjectSchema(FComponentIdGenerator& IdGenerator, UClass* Class,
 				continue;
 			}
 
+			// We do not generate a schema component if we are REP_MultiClient_InitialOnly.
```

Comment:
We're not intending on supporting dynamic components at all, so can remove the == 0 check